ouster_ros/src/os1_cloud_node.cpp

```diff
@@ -70,30 +72,32 @@ int main(int argc, char** argv) {
     CloudOS1 cloud{W, H};
     auto it = cloud.begin();
     sensor_msgs::PointCloud2 msg{};
-    double timeOffset_ms = 0.0;
-    nh.param<double>("timeOffset_ms", timeOffset_ms, 0.0);
+    double time_offset_ms = 0.0;
```

Comment:
is there a reason for `ms`? i was just thinking about using `s` since we can directly copy the output from lidar align and you don't need to convert when calling `ros::duration` later. wdyt?

Reply:
Fair point. I will keep it in SI units